### DISCUSSION TITLE
Option to detect the fan RPM of a fan at different %

### DISCUSSION THREAD

**Discussion by Rafee-M** [Rem0o/FanControl.Releases#329] [Ideas]
To get a view at what RPM a fan spins at a given %. Makes it easier to set a curve. 
For example: 
Case Fan 1:
30%: 560 RPM
50%: 840 RPM
80%: 1250 RPM

**Comment by Rafee-M** [comment]
Example: (screenshot from ASRock A-Tuning software on a ASRock B450 Steel Legend)
![image](https://user-images.githubusercontent.com/69535896/132941203-0b693ba6-1e00-4e86-9ffb-11b43a87f072.png)
![image](https://user-images.githubusercontent.com/69535896/132941328-f7a28892-478f-4893-8b94-df213915e074.png)

**Comment by Rem0o** [comment]
In the current UI, where would you fit that information?

**Reply by Jonny11bonk** [reply]
I found some problem with the graph making, making the straight line up or down is impossible. I always had to fire up the Notepad++ and do it myself.

**Reply by Rem0o** [reply]
That's by design.

**Comment by Rafee-M** [comment]
![image](https://user-images.githubusercontent.com/69535896/135252194-578d8348-23eb-4972-8158-2cd449b1ce29.png)

Maybe on the right of the "Graph" fan curve configurator or a button there that opens a pop-up window showing the fan rpm (it could be floating/appear on the right of the "Graph" window)

**Comment by Rem0o** [comment]
@RafeeDaBoy  Which fan should it show? A fan curve is not bound to a specific fan, it can be assigned to any (many) fans at the same time. What about other types of fan curves? They don't get to see the RPMs?

**Reply by Rafee-M** [reply]
Huh, in that case I'd suggest a separate card showing the fan RPM and a small drop down menu like this for selecting which fan's RPMs to display
![image](https://user-images.githubusercontent.com/69535896/135309648-f2f441c0-f417-45b6-bb34-eda3ff1cea38.png)

**Comment by MinciuPL** [comment]
Hello, i crate account to schere my proposition. 
Crate another Curve, flat etc with just RPM target. 

Like it is in Corsair ecosystem. There i coud have 3 difrent fans, with difrent rpm range. But i set up curve for RPM ant it works fine. 

Evry one ouptut was try to reach define RPM in curve. And Controller did it. If Fan have more easy airflow, and we set 1000rpm it have 1000prm lets say on 50%. Same fan in other place where have more dificuoult airflow and static preasuer block a little fan speed at 50%, then controler put 53% to rach 1000rpm.

Its easy. If Corsair dev can did it, You can do it better ;) 

This is last missing option in FanControl software. 

Next step will by use microphone from PC to find and mach curve by dB. Then it will by briliant :D

**Comment by Rem0o** [comment]
Fan calibration covers this.

**Reply by Rafee-M** [reply]
Oh yes, it does. Thanks a lot for it! Really helps in getting the perfect fan curve
